src/main/java/TobyBot.java

```diff
@@ -1,10 +1,100 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class TobyBot {
+    private static final String LINE = "____________________________________________________________";
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+
     public static void main(String[] args) {
-        System.out.println("____________________________________________________________");
+        Scanner scanner = new Scanner(System.in);
+        System.out.println(LINE);
         System.out.println("Hello! I'm TobyBot");
         System.out.println("What can I do for you?");
-        System.out.println("____________________________________________________________");
-        System.out.println("Bye. Hope to see you again soon!");
-        System.out.println("____________________________________________________________");
+        System.out.println(LINE);
```

Comment:
Keeping constant LINE is a good idea but I think I would prefer to create a method that prints the line separation